openvino/conditional_compilation/CMakeLists.txt

```diff
@@ -36,8 +36,16 @@ if(SELECTIVE_BUILD STREQUAL "COLLECT")
     FetchContent_GetProperties(ext_seapi)
     if(NOT ext_seapi_POPULATED)
         FetchContent_Populate(ext_seapi)
+        if (X86_64)
+            set (SEAPI_FORCE_BITS 64)
+        elseif(X86)
+            set (SEAPI_FORCE_BITS 32)
+        else()
+            message(FATAL_ERROR "Failed to detect target architecture")
+        endif()
         add_custom_target(build-seapi
-            COMMAND pushd ${ext_seapi_SOURCE_DIR} && python ${ext_seapi_SOURCE_DIR}/buildall.py -i && popd
+            COMMAND python ${ext_seapi_SOURCE_DIR}/buildall.py -i --force_bits=${SEAPI_FORCE_BITS}
```

Comment:
I think we can remove `-i` flag too.

Reply:
This PR is dedicated to fixing existing Job failure and is pretty urgent. Windows validation is not yet setup and I suggest we open a separate PR